test/sql/copy/csv/test_ignore_errors.test

```diff
@@ -0,0 +1,76 @@
+# name: test/sql/copy/csv/test_ignore_errors.test
+# description: IGNORE_ERRORS test
+# group: [csv]
+
+statement ok
+CREATE TABLE integers(i INTEGER, j INTEGER);
+
+statement ok
+COPY integers FROM 'test/sql/copy/csv/data/test/error_too_little.csv' (HEADER, IGNORE_ERRORS)
```

Comment:
Could we verify that these tests break (`statement error`) in case the `IGNORE_ERRORS` is not specified?